### PR TITLE
Fix dummy data

### PR DESCRIPTION
**Pull request by thejoefriel**
#### Issues this PR relates to
#417 

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: server/database/prod-data/users.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+};
+
+const createMember2User = async (member2Credentials) => {
+  const memberData = [MEMBER, member2Credentials.id];
+  const sql = `INSERT INTO users(role, credential_id) VALUES ($1, $2) RETURNING *`;
+  const res = await query(sql, memberData);
+  return res.rows[0];
+};
+
+const createEarnGroupUser = async (earnGroupCredentials) => {
```
Comment (RamyAlshurafa) [inline server/database/prod-data/users.js:27]: deleted this

File: server/database/prod-data/users.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  const earnGroupData = [EARN_GROUP, earnGroupCredentials.id];
+  const sql = `INSERT INTO users(role, credential_id) VALUES ($1, $2) RETURNING *`;
+  const res = await query(sql, earnGroupData);
+
+  return res.rows[0];
+};
+
```
Comment (RamyAlshurafa) [inline server/database/prod-data/users.js:34]: this as well

File: server/database/prod-data/users.js (hunk 1 above)
Comment (thejoefriel) [reply server/database/prod-data/users.js:12]: sorted

File: server/database/prod-data/users.js (hunk 2 above)
Comment (thejoefriel) [reply server/database/prod-data/users.js:19]: sorted

**Pull request merged**